src/main/java/org/openstreetmap/atlas/utilities/arrays/LongArray.java

```diff
@@ -1,11 +1,16 @@
 package org.openstreetmap.atlas.utilities.arrays;
 
+import org.openstreetmap.atlas.proto.ProtoSerializable;
+import org.openstreetmap.atlas.proto.adapters.ProtoAdapter;
+import org.openstreetmap.atlas.proto.adapters.ProtoLongArrayAdapter;
+
 /**
  * {@link LargeArray} for type {@link Long}
  *
  * @author matthieun
+ * @author lcram
```

Comment:
Nit: Not sure some of these changes warrant an author update.

Reply:
I thought a bit about this. Where should the line be? My internal thought was if I added at least one new method, or significantly changed a method, I added an author tag - if anything to document from whom the code in the class is coming from. 

Looking at LongArray, I think this is probably a case where I was too overzealous with the tag.  But I think for the classes where I started adding new interfaces, including equals() and hashCode() implementations, that tag is a good idea for future users of the class.

Reply:
This is a tricky area - everyone has different opinions on what's the right convention. This is only my opinion, so take that for what it's worth :) - I try to answer both of these questions with a yes before adding an author tag:
1. Did I make a change that can have downstream/user impact and/or add significant functionality on top of the existing codebase?
2. Do I feel comfortable explaining this class and its functionality and/or debugging/being responsible for any bugs that are a result of my changes?

In your case case, most of the author tags are spot on! I'd keep all of them as is, including the ones with equals/hashcode implementations. Not a fan of the fine-grained per-method author tags as that just brings more chaos to the code. I just nit-picked two classes that stood out to me as a bit overzealous (PackedAtlas and LongArray). Again, feel free to ignore :)

Reply:
Got it. I'll revert the per-method tags in a future commit. All this makes sense. Yeah I do agree that PackedAtlas and LongArray are overzealous haha.